TL improvement (#379)
07th-mod/higurashi-console-arcs: Update/omot_023.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6bd09f>茜</color>", NULL, "<color=#6bd09f>Akane</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 16, "ps3/s14/16/222200213", 256, TRUE);
	OutputLine(NULL, "「…示現流って二の太刀要らずの、上段から刀を地面に叩きつけるアレでしょ？",

> 			NULL, "\"...Jigen-ryu style? I heard they don't need to swing a sword twice because they forcefully swing it toward the ground from directly above.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"...Jigen-ryu style? Isn't that the one that swings the sword toward the ground from directly above, with enough force it doesn't require a second swing?", Line_WaitForInput);
```

//r全員が意気込んで、退出した園崎家の別室で。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "全員が意気込んで、退出した園崎家の別室で。",

> 			NULL, "Here is another room in the Sonozaki house, where all of us got enthusiastic and headed to the exit earlier.", Line_Normal);

Reviewer: ```suggestion
			NULL, "In another room of the Sonozaki house, after all of us had enthusiastically exited.", Line_Normal);
```

	ClearMessage();

//rそんなやり取りがあったことを、もちろん俺たちは知るはずもなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そんなやり取りがあったことを、もちろん俺たちは知るはずもなかった。",

> 			NULL, "Of course, we had no idea they were having such a conversation here.", Line_Normal);

Reviewer: ```suggestion
			NULL, "We, of course, had no idea they were having such an exchange.", Line_Normal);
```
